src/presets/ec/src/ec-default.scss

```
@@ -58,17 +58,17 @@
   @extend %ecl-paragraph;
 }
 
-.ecl ul:not([class*='ecl-'], [class*='wt-']),
+.ecl ul:not([class*='ecl-'], [class*='wt-'], [class*='highcharts-*']),
```

I don't think we need the second '*'. The rule will already target any class starting with 'highcharts-'
```suggestion
.ecl ul:not([class*='ecl-'], [class*='wt-'], [class*='highcharts-']),
```